docs/reference/overrides/select-value.json

```diff
@@ -2,7 +2,7 @@
   "name": "SelectValue",
   "props": {
     "children": {
-      "type": "React.ReactNode | (value) => React.ReactNode",
+      "type": "null | (value) => React.ReactNode",
       "description": "Override children if you need to customize how the value is displayed.\nBy default, renders the current value as is.\n\nAccepts a `ReactNode` or a function that returns the node to render."
```

Comment:
This override actually needed to be updated in several ways:

- `label` was always first, not `value`
- In new Base UI versions, `value`  is available now but comes second

This is the current source of truth:

```jsx
children?: null | ((label: string, value: any) => React.ReactNode);
```

Reply:
What would be a label in this case? the placeholder?

Reply:
`label` is the raw `textContent` of `Select.Item` as presented to the user (or `label` prop on `Select.Item`), while `value` is the unique identifier

Reply:
makes sense!